contracts/modules/RecoveryManager.sol

```diff
@@ -174,78 +152,35 @@ contract RecoveryManager is BaseModule, RelayerModule {
      */
     function cancelRecovery(BaseWallet _wallet) external onlyExecute onlyWhenRecovery(_wallet) {
         RecoveryConfig storage config = recoveryConfigs[address(_wallet)];
-        emit  RecoveryCanceled(address(_wallet), config.recovery);
+        emit RecoveryCanceled(address(_wallet), config.recovery);
         guardianStorage.setLock(_wallet, 0);
         delete recoveryConfigs[address(_wallet)];
     }
 
-    /**
-    * @dev Gets the details of the ongoing recovery procedure if any.
-    * @param _wallet The target wallet.
-    */
-    function getRecovery(BaseWallet _wallet) public view returns(address _address, uint64 _executeAfter, uint32 _guardianCount) {
-        RecoveryConfig storage config = recoveryConfigs[address(_wallet)];
-        return (config.recovery, config.executeAfter, config.guardianCount);
-    }
-
     /**
      * @dev Lets the owner start the execution of the ownership transfer procedure.
      * Once triggered the ownership transfer is pending for the security period before it can
      * be finalised.
      * @param _wallet The target wallet.
      * @param _newOwner The address to which ownership should be transferred.
      */
-    function executeOwnershipTransfer(
-        BaseWallet _wallet,
-        address _newOwner
-    )
-        external
-        onlyWalletOwner(_wallet)
-        onlyWhenUnlocked(_wallet)
-        notWhenOwnershipTransfer(_wallet)
+    function transferOwnership(BaseWallet _wallet, address _newOwner) external
+    onlyExecute
+    onlyWhenUnlocked(_wallet)
```

Comment:
I do agree with the style guide for handling long function declaration with modifiers:
>If a long function declaration has modifiers, then each modifier should be dropped to its own line

Currently `ethlint` limits the line length to 145 chars, which this function does not exceed so it's not considered "long" per se. 

My suggestion is to always drop modifiers on own line and only drop function parameters on own line if it exceeds 145 chars or there are more than 5 parameters. The latter is also an `ethlint` rule which shows as a warning if not kept: 

`warning    In case of more than 5 parameters, drop each into its own line.    arg-overflow`

Reply:
Can we not do that? This is just a matter of convention and personal taste, but always dropping the modifiers on their own line is a convention we are not using in the rest of the code and I personally don't like it.

Can we agree to drop the parameters and the modifiers on their own line if the line length is 145 or more and if there are more than 5 parameters?